modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc

```diff
@@ -5,28 +5,30 @@
 +
 Red Hat recommends that you recreate any existing workspaces using {code-ready-name-short} {code-ready-version}.
 
-* You can now configure AMQ Online.
+* In {amq-online-name-short}, rejected messages are retried indefinitely instead of being sent to the Dead Letter Queue.
+
+* You can now configure {amq-online-name-short}.
 +
 --
 :cluster-administrator: cluster administrator
 
-If you plan to use AMQ Online in {PRODUCT_SHORT}, you need to create an AMQ Online configuration for production. 
-You must edit and apply YAML files using the command line (CLI) tools to configure AMQ Online. 
+If you plan to use {amq-online-name-short} in {PRODUCT_SHORT}, you need to create an {amq-online-name-short} configuration for production. 
+You must edit and apply YAML files using the command line (CLI) tools to configure {amq-online-name-short}. 
 
-This section provides general guidance on configuring AMQ Online for production usage in {PRODUCT_SHORT}.
+This section provides general guidance on configuring {amq-online-name-short} for production usage in {PRODUCT_SHORT}.
 
 .Prerequisites
 * You are a {cluster-administrator} with the OpenShift CLI (`oc`) installed locally.
```

Comment:
Separate into two bullet entries. 
You are  a cluster....
You have CLI installed